File: Demo_CS/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Demo_CS
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            this.expTree1.StartUpDirectory = ExpTreeLib.ExpTree.StartDir.Desktop;
            this.expList1.ViewType = (int)View.LargeIcon;
        }

        //Load files to ExpFileList
        private void expTree1_ExpTreeNodeSelected(string SelPath, ExpTreeLib.CShItem Item)
        {
            bool includeFolder = true;
            this.expList1.DisplayFiles(SelPath, Item, includeFolder);
        }

        private void expList1_ExpListItemDoubleClick(string SelPath, ExpTreeLib.CShItem Item)
        {
            this.expTree1.ExpandANode(Item);
        }
    }
}

File: Demo_VB/Form1.vb
Public Class Form1
    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        Me.ExpTree1.StartUpDirectory = ExpTreeLib.ExpTree.StartDir.Desktop
        Me.ExpList1.ViewType = View.LargeIcon
    End Sub

    'Load files to ExpFileList
    Private Sub ExpTree1_ExpTreeNodeSelected(SelPath As String, Item As ExpTreeLib.CShItem) Handles ExpTree1.ExpTreeNodeSelected
        Dim includeFolder As Boolean = True
        Me.ExpList1.DisplayFiles(SelPath, Item, includeFolder)
    End Sub

    Private Sub ExpList1_ExpListItemDoubleClick(SelPath As String, Item As ExpTreeLib.CShItem) Handles ExpList1.ExpListItemDoubleClick
        Me.ExpTree1.ExpandANode(Item)
    End Sub
End Class
